third_party/glog/BUILD

```diff
@@ -40,36 +56,20 @@ genrule(
             "//third_party/libunwind:libunwind-files",
          ]
     }),
-    outs = [
-        "include/glog/log_severity.h",
-        "include/glog/logging.h",
-        "include/glog/raw_logging.h",
-        "include/glog/stl_logging.h",
-        "include/glog/vlog_is_on.h",
-        "lib/libglog.a",
-    ],
+    outs = include_files + lib_files,
     cmd = select({
-        "//tools/platform:darwin": DARWIN_EXECUTE,
-        "//conditions:default": LINUX_EXECUTE,
+        "//tools/platform:darwin": mac_script,
+        "//conditions:default": linux_script,
     }),
 )
 
 cc_library(
     name = "glog-cxx",
-    srcs = [
-        "empty.cc",
-        "lib/libglog.a",
-        "include/glog/log_severity.h",
-        "include/glog/raw_logging.h",
-        "include/glog/stl_logging.h",
-        "include/glog/vlog_is_on.h",
-    ],
+    srcs = ["empty.cc"] + lib_files,
```

Comment:
The include files are in the hdrs now. Realized this is not needed in the srcs actually.